src/Core/Billing/Services/Implementations/SubscriberService.cs

```diff
@@ -137,6 +138,76 @@ public class SubscriberService(
         }
     }
 
+    public async Task<Customer> GetCustomerOrThrow(
```

Comment:
Anything in here related to `GetCustomerOrThrow` is just me re-alphabetizing the methods in this service.